src/askui/models/anthropic/settings.py

```diff
@@ -28,4 +30,7 @@ class ClaudeSettings(ClaudeSettingsBase):
 
 
 class ClaudeComputerAgentSettings(ComputerAgentSettingsBase, ClaudeSettingsBase):
-    pass
+    thinking: ThinkingSettings | None = None
+    tool_choice: BetaToolChoiceParam | BetaToolChoiceAutoParam = (
```

Comment:
```suggestion
    tool_choice: BetaToolChoiceParam = (
```

Doesn't `BetaToolChoiceParam` include `BetaToolChoiceAutoParam`?